pytorch_lightning/utilities/__init__.py

```diff
@@ -52,3 +52,15 @@
 FLOAT16_EPSILON = numpy.finfo(numpy.float16).eps
 FLOAT32_EPSILON = numpy.finfo(numpy.float32).eps
 FLOAT64_EPSILON = numpy.finfo(numpy.float64).eps
+
+
+def _get_default_scheduler_config():
+    return {
+        'scheduler': None,
+        'name': None,  # no custom name
+        'interval': 'epoch',  # after epoch is over
+        'frequency': 1,  # every epoch/batch
+        'reduce_on_plateau': False,  # most often not ReduceLROnPlateau scheduler
+        'monitor': None,  # value to monitor for ReduceLROnPlateau
+        'strict': True,  # enforce that the monitor exists for ReduceLROnPlateau
+    }
```

Comment:
As suggested, any chance we can make this default config live in the optimizer.py class?

Reply:
There is a cyclic import and it breaks.

Reply:
Should be solved with https://github.com/PyTorchLightning/pytorch-lightning/pull/6038/commits/c51a7e533b2f2757a17ee49f33bd323108a3c339

Had to use a local import though